packages/terra-collapsible-menu-view/src/CollapsibleMenuViewHyperlink.jsx

```diff
@@ -0,0 +1,132 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import Hyperlink from 'terra-hyperlink';
+import Menu from './_CollapsibleMenu';
+import styles from './CollapsibleMenuView.module.scss';
+
+const cx = classNames.bind(styles);
+
+const variants = {
+  DEFAULT: 'default',
+  EXTERNAL: 'external',
+  IMAGE: 'image',
+  VIDEO: 'video',
+  AUDIO: 'audio',
+  DOCUMENT: 'document',
+};
+
+const contextTypes = {
+  isCollapsibleMenuItem: PropTypes.bool,
+};
```

Comment:
What is a contextType?

Reply:
It's used for checking whether the item should be displayed or truncated into the collapsed menu

Reply:
This does not seem correct.

Reply:
Same as above, the legacy API https://legacy.reactjs.org/docs/legacy-context.html#referencing-context-in-stateless-function-components

Reply:
We should not be using a legacy API.

Reply:
Discussion continued here: https://github.com/cerner/terra-framework/pull/1797#discussion_r1338751838